airflow/providers/ydb/hooks/_vendor/dbapi/cursor.py

```diff
@@ -77,11 +88,13 @@ def wrapper(*args, **kwargs):
 class Cursor:
     def __init__(
         self,
+        driver: Union[ydb.Driver, ydb.aio.Driver],
```

Comment:
This is in a `_vendor` file -- we shouldn't be making changes to it here. What version of the upstream file has these changes in?

Reply:
This was added here https://github.com/ydb-platform/ydb-sqlalchemy/commit/91a13e059c3f8a135474e181b410af901a34ab0b

Reply:
I would suggest to update readme https://github.com/apache/airflow/blob/360e727ef0ed9d151e343c497e20aa609e06e58e/airflow/providers/ydb/hooks/_vendor/readme.md

and fill in precise revision of vendor package

Reply:
> I would suggest to update readme https://github.com/apache/airflow/blob/360e727ef0ed9d151e343c497e20aa609e06e58e/airflow/providers/ydb/hooks/_vendor/readme.md
> 
> and fill in precise revision of vendor package

done